ui/app/mirrors/edit/[mirrorId]/cdcDetails.tsx

```diff
@@ -140,4 +146,78 @@ export function numberWithCommas(x: any): string {
   return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
 }
 
+function statusChangeHandle(
+  mirrorConfig: FlowConnectionConfigs,
+  mirrorStatus: FlowStatus
+) {
+  // hopefully there's a better way to do this cast
+  if (mirrorStatus.toString() === FlowStatus[FlowStatus.STATUS_RUNNING]) {
+    return (
+      <button
```

Comment:
use these instead
```
import { Button } from '@/lib/Button';
import { Icon } from '@/lib/Icon';
```

See `mirrors/create/page.tsx` as an example of Icon button.